recursive-list-main/src/components/folder/folder.component.tsx

```diff
@@ -0,0 +1,46 @@
+import { Box, Link, Paper, Typography } from "@material-ui/core";
+import { Link as RouterLink } from "react-router-dom";
+import { useFolderStyles } from "./folder.styles";
+
+import { List } from "components/list/list.component";
+
+export interface FolderProps {
+  folder: List;
+}
+
+const Folder = ({ folder }: FolderProps) => {
+  const classes = useFolderStyles();
+
+  return (
+    /* Checks if it is a file or folder and renders accordingly */
+    <div className={classes.folderContainer}>
+      {folder.isFolder ? (
+        <Link
+          style={{ textDecoration: "none" }}
```

Comment:
avoid inline styles wherever possible

Reply:
Used custom props in place of inline style